pkg/zonemgr/internal/zone-file-cache/zone_file_cache_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package zone_file_cache_test
```

Comment:
Why the suite has different package than the test?

Reply:
In the test I use some of the private data members for unit tests so I can't have it in a different package.

Reply:
Yes, please put both under zone-file-cache. The suite should be one the same.package as the test.

Reply:
fixed